drivers/adc/adc_sf32lb.c

```diff
@@ -31,6 +31,8 @@ LOG_MODULE_REGISTER(adc_sf32lb, CONFIG_ADC_LOG_LEVEL);
 #define ADC_RDATAX(n)    (ADC_RDATA + (((n) >> 1) * 4U))
 #define ADC_SLOT_REGX(n) (ADC_SLOT_REG + (n) * 4U)
 
+#define ADC_SF32LB_DEFAULT_VREF_INTERNAL 3300
```

Comment:
is it always 3.3V?

<img width="860" height="360" alt="Image" src="https://github.com/user-attachments/assets/5b50a10b-1a66-415f-9c00-c21b4102cc06" />

Reply:
checked that recommend analog AVDD always be 3.3V